modules/ting_search_carousel/ting_search_carousel.module

```diff
@@ -283,31 +283,30 @@ function _ting_search_carousel_find_entities_with_covers($query, $start, $size,
   $finished = FALSE;
   $entities = array();
 
-  $options = array(
-    'facets' => array(),
-    'enrich' => TRUE,
-    'allObjects' => FALSE,
-  );
+  $sal_query = ting_start_query()
+    ->setRawQuery($query)
+    ->setPage($start + 1)
+    ->setCount($size)
+    ->addFieldFilter('enrich', TRUE)
+    ->addFieldFilter('allObjects', FALSE)
+    ->setMaterialFilter($ignore);
```

Comment:
This does not seem right. 

1. Are `enrich` and `allObjects` actual fields?
2. According to the DocBlock `$ignore` is "Material ids to skip". `setMaterialFilter()` is material ids to include.